base/base-docset-owner-guide.md

```diff
@@ -0,0 +1,44 @@
+# {Doctype} customization guide
+
+This guide describes a {doctype} document's:
+* Unique characteristics,
+* When to include the {doctype} dotype in a project's documentation set, and
```

Comment:
**Nitpick (non-blocking:** Is `{doctype} dotype` a typo?

Reply:
Yes. Good catch. Thanks.